### PR TITLE
Share icon

### LINKED ISSUE #57: Add share button for sharing organization view link

### PR DESCRIPTION
**Pull request by CoasterFan5**
Fixes #57

### CODE REVIEW

File: src/routes/(app)/org/[id=integer]/+page.svelte [hunk 1]
```diff
@@ -48,6 +48,45 @@
...
+	const startShare = async () => {
+		let shareUrlId = data.org.slug?.slug || data.org.id;
+		let shareUrl = `${window.origin}/org/${shareUrlId}`;
+
+		try {
+			await navigator.share({
+				url: shareUrl
+			});
+			window.addEventListener('mousedown', addShareModel);
+			window.addEventListener('focus', addShareModel);
```
Comment (LeoDog896) [inline src/routes/(app)/org/[id=integer]/+page.svelte:71]: Are you sure this wont run if a person navigates to a different page on their history?

**Review by LeoDog896 — CHANGES_REQUESTED** [review]
Just a question

File: src/routes/(app)/org/[id=integer]/+page.svelte (hunk 1 above)
Comment (CoasterFan5) [reply src/routes/(app)/org/[id=integer]/+page.svelte:71]: Based on my current testing, I do not believe it will.

**Review by LeoDog896 — CHANGES_REQUESTED** [review]
run fmt, other than that looks good!

**Review by LeoDog896 — APPROVED** [review]

**Pull request merged**